php/content.md

```diff
@@ -59,7 +59,7 @@ We recommend that you add a custom `php.ini` configuration. `COPY` it into
 same commands to build and run:
```

Comment:
There's one more in this paragraph that you appear to have missed when you copied the change over. :+1: